### PR TITLE
Update SUPPORTED_LANGUAGES.md

### PR DESCRIPTION
**Pull request by laurelthorburn**
~~Awaiting creation of liquid repo~~

I have submitted a request to add liquid to highlightjs.

I am now following the after steps listed in the todo:

https://github.com/highlightjs/highlight.js/issues/4173

New repo: https://github.com/highlightjs/highlightjs-liquid

### Changes
Adds liquid to the supported languages table.

### Checklist
- [x] Added markup tests, or they don't apply here because... proposing new supported language
- [x] Updated the changelog at `CHANGES.md`

Related to historical issue: https://github.com/highlightjs/highlight.js/issues/1055

### CODE REVIEW

File: SUPPORTED_LANGUAGES.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ The table below shows the full list of languages (and corresponding classes/alia
 | Lasso                   | lasso, ls, lassoscript |         |
 | Less                    | less                   |         |
 | LDIF                    | ldif                   |         |
+| Liquid                  | liquid                 | [highlightjs-liquid](https://github.com/leanprover-community/highlightjs-liquid) |
```
Comment (joshgoebel) [inline SUPPORTED_LANGUAGES.md:129]: Reason this wouldn't link to the new repo here?

**Comment by Yang062** [comment]
Josh Goebel ***@***.***> 于 2024年11月30日周六 上午9:51写道：

File: SUPPORTED_LANGUAGES.md (hunk 1 above)
Comment (laurelthorburn) [reply SUPPORTED_LANGUAGES.md:129]: ugh I thought I swapped the link, that was a placeholder.

**Review by joshgoebel — APPROVED** [review]

**Comment by laurelthorburn** [comment]
@joshgoebel Is there anything I need to do to get Liquid to show in the demo?

https://highlightjs.org/demo

**Comment by joshgoebel** [comment]
Only 1st party grammars show up in the demo.

**Pull request merged**